src/httpserver.cpp

```diff
@@ -9,6 +9,7 @@
 #include <compat.h>
 #include <util.h>
 #include <utilstrencodings.h>
+#include <utilthreadnames.h>
```

Comment:
should be moved into util folder depending on order of merges

Reply:
Would suggest merging the refactor first, then I'll replay this PR on top of it and put it in the util directory